app/models/conversion_host/configurations.rb

```diff
@@ -54,6 +54,9 @@ def enable_queue(params, auth_user = nil)
       params = params.symbolize_keys
       resource = params.delete(:resource)
 
+      conversion_host = ConversionHost.where(:resource => resource)
+      raise "the resource '#{resource.name}' is already configured as a conversion host" unless conversion_host.empty?
```

Comment:
Minor, but it's more efficient to do:

`raise "blah" unless ConversionHost.exists?(:resource => resource)`